### ISSUE TITLE
OPTIONS for /.well-known/webcrypto-socket missing required CORS header

### ISSUE THREAD

**Issue by WorldThirteen** [PeculiarVentures/fortify#346]
Request: OPTIONS https://127.0.0.1:31337/.well-known/webcrypto-socket doesn't contain 'Access-Control-Allow-Origin': '*' header, while GET for https://127.0.0.1:31337/.well-known/webcrypto-socket does.

This leads to a case when the web app couldn't connect to Fortify.

Since there is a case when a browser will send an OPTIONS preflight request even for GET request, it is required to have such a response header for OPTIONS request too.

**Comment by microshine** [comment]
What browser does throw that exception (and version)?

**Comment by donskov** [comment]
Chrome 84

**Comment by donskov** [comment]
@microshine You can call `fetch` for catch the error:

```js
fetch('https://127.0.0.1:31337/.well-known/webcrypto-socket', { method: 'OPTIONS' })
```

**Comment by microshine** [comment]
https://github.com/PeculiarVentures/fortify/releases/tag/v1.3.6

**Comment by WorldThirteen** [comment]
Fixed in 1.3.6.
